### DISCUSSION TITLE
Implementing data-grid search option in ng-matero

### DISCUSSION THREAD

**Discussion by rahulpd101** [ng-matero/ng-matero#590] [Q&A]
Hi,
I implemented a data-grid using ng-matero. I am unable to implement the search functionality in mtx-grid by referencing angular material table datasource or even manually.
Please help me out here.

**Comment by nzbin** [comment]
Can you give me an online example?

**Reply by rahulpd101** [reply]
### I would have to write the whole table with random data on stackblitz. Can we schedule a meet if possible? (Google meet/amazon chime) anything alike them.
If YES, then please mention your timezone, I'll schedule a meeting which will suite your available hours.
